sdk/python/kfp/compiler/compiler.py

```diff
@@ -30,7 +30,7 @@
 
 from ._default_transformers import add_pod_env
 
-from ..components._structures import InputSpec
+from ..components.structures import InputSpec
```

Comment:
I remember all the module under components has underscore prefix. Did they changed recently?

Reply:
yes, see https://github.com/kubeflow/pipelines/commit/c83aff2738f77778802fa9fb034bcacc233d125e